core/src/gl/texture.cpp

```diff
@@ -8,6 +8,14 @@
 #include "tangram.h"
 #include "log.h"
 
+// Only JPEG, PNG and GIF
+#define STBI_NO_BMP
+#define STBI_NO_PSD
+#define STBI_NO_TGA
```

Comment:
From discussion in slack: let's keep PSD and TGA  and remove BMP, HDR, PIC, and PNM.